docs/nuvla/user-guide/customize-apps.md

```diff
@@ -77,20 +78,11 @@ Very frequently a running container must pass some information back
 to the user. A container generating a random password or token for
 accessing services within a container is a typical example. 
 
-To do this, you can provide a list of output parameters when you
-define an application. Values for these output parameters can then be
-set by the container when it runs. Setting the values of output
-parameters can be done most easily via the Nuvla Python API. See below
-for details.
+To do this, you can provide a list of output parameters when you define an application. Values for these output parameters can then be set by the container when it runs. Setting the values of output parameters can be done most easily via the Nuvla Python client API. See below for details.
 
 ## Uploading Containers
 
-Nuvla can use any image that has been uploaded to any open Docker
-repository. (Repositories that require credentials are not yet
-supported.) The easiest to use is Docker Hub.  If you have an account
-on Docker Hub, you can create an organization to hold your
-"repositories", each of which holds multiple tagged versions of an
-image.
+Nuvla can use any image that has been uploaded to any Docker repository. By default, Nuvla expects Docker Hub.  If you have an account on Docker Hub, you can create an organization to hold your "repositories", each of which holds multiple tagged versions of an image.
```

Comment:
In fact it's not Nuvla which uses by default the Docker Hub but Docker itself.

Reply:
Ok thanks